extensions/wikia/AdEngine/js/context/slotsContext.js

```diff
@@ -26,12 +25,13 @@ define('ext.wikia.adEngine.context.slotsContext', [
 
 	function setupSlots() {
 		var context = adContext.getContext(),
-			isHome = params.getPageType() === 'home',
+			isHome = context.targeting.pageType === 'home',
 			isOasis = context.targeting.skin === 'oasis',
 			isPremiumAdLayoutEnabled = context.opts.premiumAdLayoutEnabled,
 			isIncontentEnabled =
 				!isHome &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)